Algorithms/Medium/287_FindTheDuplicateNumber/Solution.cpp

```diff
@@ -0,0 +1,16 @@
+class Solution {
+public:
+    int findDuplicate(vector<int>& nums) {
+        sort(nums.begin(),nums.end());
+        int n =nums.size();
+        for(int i = 0;i<n-1;i++)
+        {
+            if(nums[i] == nums[i+1])
+                return nums[i];
+        }
+        return -1;
+    }
+};
+
+// Time Complexity: O(n)
```

Comment:
Your solution's time complexity is O(nlogn) because of calling the sort function.

Reply:
Oh, yes, thank you. I will do make changes.